about us/about_us.css

```diff
@@ -65,6 +64,86 @@ input[type="checkbox"] {
   border: 2px solid black;
 }
 
+.known-obstacles {
+  display: flex;
+  justify-content: space-evenly;
+  text-indent: 0px;
```

Comment:
why do you need these text indents?

Reply:
bcs I have it for the whole body and don't need them in certain parts